activemodel/lib/active_model/validations/acceptance.rb

```diff
@@ -3,12 +3,12 @@ module ActiveModel
   module Validations
     class AcceptanceValidator < EachValidator # :nodoc:
       def initialize(options)
-        super({ allow_nil: true, accept: "1" }.merge!(options))
+        super({ allow_nil: true, accept: ["1", true] }.merge!(options))
```

Comment:
@kuldeepaggarwal - the point is now you can specify an array of acceptable values.

If people were using 'true' before their tests weren't passing, so that's a non issue.  Now you will be able to accept `["1", true, "true"]` as you suggest by configuring it in the class:

```
class User < ActiveRecord::Base
  validates_acceptance_of :terms_and_conditions, accept: ["1", true, "true"]
end
```

You can have it accept anything in the array where before an array wasn't possible, just a single value.